### DISCUSSION TITLE
Quick actions

### DISCUSSION THREAD

**Discussion by leonardsellem** [LeoKlaus/Paperparrot#204] [Q&A]
Hey,
It looks like quick actions have been released with v1.8 back in Oct '23, but I can't find how to turn them on. Any hint ?

**Comment by LeoKlaus** [comment]
Hey Leonard,
I believe by quick actions you mean the shortcuts on the home screen?
First off, these are not available on Mac. On iOS, there's no specific way to disable/enable them, they should just be active by default.
You should be able to see them by long-pressing on the Paperparrot icon on your home screen.

If they're not available I'd suggest trying to restart your phone/iPad and see if that helps.
